book/MCDM Productions; Arcadia Issue 8.json

```
@@ -0,0 +1,3593 @@
+            "subclassShortName": "Hag",
+            "subclassSource": "Ar8",
+            "page": 4,
+            "level": 1,
+            "entries": [
+                "Choose your patron’s kind or determine it randomly, rolling on the Hag Kind table below. If your patron is a coven of hags, you gain the coven features of this subclass, regardless of the kind of hags that make up the coven.",
+                {
+                    "type": "table",
+                    "caption": "Hag Kind",
+                    "colLabels": [
+                        "d4",
+                        "Kind"
+                    ],
+                    "colStyles": [
+                        "col-2 text-center",
+                        "col-2 text-center"
+                    ],
+                    "rows": [
+                        [
+                            "1",
+                            "Green"
+                        ],
+                        [
+                            "2",
+                            "Night"
+                        ],
+                        [
+                            "3",
+                            "Sea"
+                        ],
+                        [
+                            "4",
+                            "Coven"
+                        ]
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "Expanded Spell List",
+            "source": "Ar8",
+            "className": "Warlock",
+            "classSource": "PHB",
+            "subclassShortName": "Hag",
+            "subclassSource": "Ar8",
+            "page": 5,
+            "level": 1,
+            "entries": [
+                "The Hag lets you choose from an expanded list of spells when you learn a warlock spell. The following spells are added to the warlock spell list for you, determined by your patron’s kind: green, night, sea, or coven.",
+                {
+                    "type": "table",
+                    "colLabels": [
+                        "Spell Level",
+                        "Green Hag Spells",
+                        "Night Hag Spells",
+                        "Sea Hag Spells",
+                        "Coven Spells"
+                    ],
+                    "colStyles": [
+                        "col-2 text-center",
+                        "col-2 text-center",
+                        "col-2 text-center",
+                        "col-2 text-center",
+                        "col-2 text-center"
+                    ],
+                    "rows": [
+                        [
+                            "1st",
+                            "{@spell detect magic}, {@spell silent image}",
+                            "{@spell magic missile}, {@spell sleep}",
+                            "{@spell create or destroy water}, {@spell fog cloud}",
+                            "{@spell animal friendship}, {@spell tasha's hideous laughter|hideous laughter}"
```

With `@spell` tags, the pipe order is `name|source|optional text`. So as you had it, you were linking to the `tasha's...` spell with the `hideous laughter` source.

(Be aware that spell tags default to the `PHB` source by default, so doing `{@spell tasha's hideous laughter||hideous laughter` is also possible. I just wrote it to make it more clear.
```suggestion
                            "{@spell animal friendship}, {@spell tasha's hideous laughter|PHB|hideous laughter}"
```